internal/network/vds_subresource.go

```diff
@@ -0,0 +1,96 @@
+/*
+ *  Copyright 2023 VMware, Inc.
+ *    SPDX-License-Identifier: MPL-2.0
+ */
+
+package network
+
+import (
+	"fmt"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+	validation_utils "github.com/vmware/terraform-provider-vcf/internal/validation"
+	"github.com/vmware/vcf-sdk-go/models"
+)
+
+// VdsSchema this helper function extracts the VDS Schema, so that
+// it's made available for both Domain and Cluster creation.
+// This specification contains vSphere distributed switch configurations.
+func VdsSchema() *schema.Resource {
+	return &schema.Resource{
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:         schema.TypeString,
+				Required:     true,
+				Description:  "vSphere Distributed Switch name",
+				ValidateFunc: validation.NoZeroValues,
+			},
+			"is_used_by_nsx": {
+				Type:        schema.TypeBool,
+				Optional:    true,
+				Description: "Identifies if the vSphere distributed switch is used by NSX-T",
```

Comment:
```suggestion
				Description: "Identifies if the vSphere distributed switch is used by NSX",
```

Product was renamed.